ddtrace/span.py

```diff
@@ -474,7 +474,10 @@ def set_exc_info(self, exc_type, exc_val, exc_tb):
         # readable version of type (e.g. exceptions.ZeroDivisionError)
         exc_type_str = "%s.%s" % (exc_type.__module__, exc_type.__name__)
 
-        self.meta[ERROR_MSG] = str(exc_val)
+        if isinstance(exc_val, bytes):
+            self.meta[ERROR_MSG] = exc_val.decode()
+        else:
+            self.meta[ERROR_MSG] = stringify(exc_val)
```

Comment:
```suggestion
        self.meta[ERROR_MSG] = ensure_text(exc_val)
```
where `ensure_text` comes from `ddtrace.internal.compat`.